app/models/trip.rb

```diff
@@ -0,0 +1,48 @@
+class Trip < ApplicationRecord
+  belongs_to :passenger
+  belongs_to :driver
```

Comment:
no validations on trip's cost or date. You handled it on the site so that it doesn't produce any errors, but I just wanted to bring that up